sdk/identity/identity/src/credentials/clientSecretCredential.ts

```diff
@@ -1,28 +1,29 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import qs from "qs";
```

Comment:
Just to confirm, since I don't seem to be able to match this anywhere else: Don't we need to bundle "qs" as part of the browser build?

Reply:
It already gets bundled as a result of being imported in this file, I've verified the browser build output to make sure.